contribute/style-guides/documentation-style-guide.md

```diff
@@ -115,6 +115,34 @@ When possible, use the exact title of the page or section you are linking to as
 **Example**
 Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
 
+### Notes, tips, cautions, and warnings
+
+Grafana documentation uses notes, tips, cautions, and warnings. Notes are the most common. The format for all of them is indented, bold, sentence case:
+
+```
+> **Note:**
+```
+
+#### Notes
+
+Notes are for additional information that the user should be extra aware of. For example:
+
+> **Note:** This page describes a feature for Grafana 7.0 beta.
+
+#### Tips
+
+Tips are alternate or more efficient ways of doing things. Rarely used.
+
+#### Cautions
```

Comment:
Should be Caution?

Reply:
No. I made Cautions and Warnings plural to maintain parellelism with Notes and Tips.